### ISSUE TITLE
is there a plan to move to RxJava 2.0?

### ISSUE THREAD

**Issue by SvenDowideit** [amsa-code/risky#3]
is it going to be painful?

**Comment by davidmoten** [comment]
Yep at some stage especially as RxJava 1.0 has just gone end-of-life. There is a goodly amount of work in it but we'll get there. I'll pop it in our JIRA backlog at work.

**Comment by davidmoten** [comment]
By the way there is nothing stopping you using RxJava 1.0 and 2.0 at the same time as there is an [interop library](https://github.com/akarnokd/RxJava2Interop) (the APIs occupy different package namespaces).

**Comment by SvenDowideit** [comment]
interesting :) this is my first time in ~20 years doing Java, so for me its more relevant only because its hard to navigate examples out on the web, and the differences between versions.
